modules/workbench/modules/workload_identity/main.tf

```diff
@@ -0,0 +1,40 @@
+# adapted from https://github.com/jtreutel/circleci-gcp-oidc-terraform/blob/master/main.tf
+
+resource "google_iam_workload_identity_pool" "circleci" {
+  project = var.project_id
+  workload_identity_pool_id = "circleci-wip-${var.aou_env}"
+  display_name              = "CircleCI WIP - ${var.aou_env}"
```

Comment:
do we need a WIP for each env? I throught we just need WIP for test?

Reply:
This will eventually enable deployment in all environments

Reply:
fine for now. I am a bit concern for higher env. The deploy SA in prod and preprod has permission to accessing files across perimter (it's currently also used in cdr publish).  We might need to be careful to protect those deploy CI...